src/index.js

```diff
@@ -0,0 +1,237 @@
+    });
+};
+
+const displayRealTemp = (tempF) => {
+  const tempValueElement = document.getElementById('tempValue');
+  // tempValueElement.textContent = state.tempF;
+  tempValueElement.textContent = tempF;
+};
+
+// Function to increase temp Value by clicking up arrow
+const incTempAction = () => {
+  state.tempValue += 1;
+  const tempDigit = document.getElementById('tempValue');
+  tempDigit.textContent = state.tempValue;
+  tempValueColorAction();
+  landscapeChangeAction();
+};
+
+// Function to decrease temp Value by clicking up arrow
+const decTempAction = () => {
+  state.tempValue -= 1;
+  const tempDigit = document.getElementById('tempValue');
+  tempDigit.textContent = state.tempValue;
+  tempValueColorAction();
+  landscapeChangeAction();
+};
+
+// Function to change color of temp value according to temp range
+const tempValueColorAction = () => {
+  console.log('tempValueColor being called');
+  // document.getElementById('tempValue').style.color = 'orange';
+  // state.color = 'orange';
+  const tempColorElement = document.getElementById('tempValue');
+  // tempColorElement.style.color = 'orange';
+  // tempColorElement.className = color;
+  if (state.tempValue <= 49) {
+    tempColorElement.style.color = 'teal';
+  } else if (50 <= state.tempValue && state.tempValue <= 59) {
+    tempColorElement.style.color = 'green';
+  } else if (60 <= state.tempValue && state.tempValue <= 69) {
+    tempColorElement.style.color = 'yellow';
+  } else if (70 <= state.tempValue && state.tempValue <= 79) {
+    tempColorElement.style.color = 'orange';
+  } else {
+    tempColorElement.style.color = 'red';
+  }
+};
+
+// Function to change landscape of temp value according to temp range
+const landscapeChangeAction = () => {
+  const landscapeElement = document.getElementById('landscape');
+  landscapeElement.textContent = state.landscape;
+
+  if (state.tempValue <= 59) {
+    landscapeElement.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    state.background = "url('../assets/winter.jpg')";
+    document.querySelector('html').style.backgroundImage =
+      "url('../assets/winter.jpg')";
+  } else if (60 <= state.tempValue && state.tempValue <= 69) {
+    landscapeElement.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    state.background = "url('../assets/autumn.jpg')";
+    document.querySelector('html').style.backgroundImage =
+      "url('../assets/autumn.jpg')";
+  } else if (70 <= state.tempValue && state.tempValue <= 79) {
+    landscapeElement.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    state.background = "url('../assets/summer.jpg')";
+    document.querySelector('html').style.backgroundImage =
+      "url('../assets/summer.jpg')";
+  } else if (80 <= state.tempValue) {
+    landscapeElement.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    // state.background = url('../assets/desert.jpg');
+    document.querySelector('html').style.backgroundImage =
+      "url('../assets/desert.jpg')";
+    // document.html.style.backgroundImage = "url('../assets/desert.jpg')";
+  }
```

Comment:
Check out the logic here, see how similar the branches are of the conditional statement. We look for whether the temperature is within a range, and pick a color accordingly. What if we had a list of objects that we could iterate through to find the values. We could set up something like

```javascript
const TEMP_LANDSCAPE = [ 
  { upperBound: 59, landscape: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲', background: 'url('../assets/winter.jpg')'}, 
  { upperBound: 69, landscape: '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃', background: 'url('../assets/autumn.jpg')'} 
];
```

Then your method would iterate through `TEMP_LANDSCAPE` and find the first record that has an upper bound higher than our temperature, then use it as the source of picking the landscape which would help make the function a bit more concise.

This could accommodate a scenario where you might have even more landscapes in the future, which would prevent your conditional statement from being a really long block of if/elif/elif and so on